packages/expo-cli/src/credentials/context.ts

```diff
@@ -64,6 +70,9 @@ export class Context {
       const { exp } = getConfig(projectDir);
       this._manifest = exp;
       this._hasProjectContext = true;
+      this._iosApiClient = new IosApi(this.user).withProjectContext(this);
```

Comment:
could use some more context on what's going on here - why do create an api client if the status is fatal? and why not with the project context?

Reply:
i updated this with better comments -- basically we use `Doctor.validateWithoutNetworkAsync` to check whether there is a manifest in the project directory. If there is (we get a non fatal response), we initialize the necesary project context things, else we do a normal initialization